R/eml_data_qa.R

```diff
@@ -73,8 +71,7 @@ qa_package <- function(node, pid, readAllData = TRUE,
 
     # Checks that each data object has a matching url in the eml.
     wrong_URL <- FALSE
-    for (i in seq_along(package$data)) {
-        data <- package$data[i] # use this as opposed to (data in package$data) to preserve names
+    for (data in package$data) {
```

Comment:
Can we call this `datapid` or something similar for clarity? I use `data` for the actual dataframe of the data.

Reply:
Yes, will fix